docs/queries/byrole.mdx

```diff
@@ -314,3 +315,36 @@ To learn more about the `aria-level` property, see
 
 > The `level` option is _only_ applicable to the `heading` role. An error will
 > be thrown when used with any other role.
+
+### `description`
+
+You can filter the returned elements by their
+[accessible description](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_description)
+for those cases where you have several elements with the same role and they
+don't have an accessible name but they do have a description.<br /> This would
+be the case for elements with
+[alertdialog](https://www.w3.org/TR/wai-aria-1.1/#alertdialog) role, where the
+`aria-describedby` attribute is used to describe the elements content.
+
+For example in
+
+```html
+<body>
+  <ul>
+    <li role="alertdialog" aria-describedby="notification-id-1">
+      <div><button>Close</button></div>
+      <div id="notification-id-1">You have unread emails</div>
+    </li>
+    <li role="alertdialog" aria-describedby="notification-id-2">
+      <div><button>Close</button></div>
+      <div id="notification-id-2">Your session is about to expire</div>
+    </li>
+  </ul>
+</body>
+```
+
+You can query an specific element like this
```

Comment:
```suggestion
You can query the specific element like this
```

Reply:
In this case, I think the right wording would be
> You can query a specific element like this

since we want the user know he can select any specific element of those which have the same role.

Do you think we should actually use "the" instead?

(updated in this commit: 9da0a1598132cb4c75ffa91d03ab9f16f24a9784)